### ISSUE TITLE
Scripts for how the pretrained models where trained?

### ISSUE THREAD

**Issue by jzju** [UKPLab/sentence-transformers#1576]
I would like to recreate paraphrase-multilingual-mpnet-base-v2 but with only a few lagnuages and see if performance can improve. Could you publish end to end scripts including data fetching that generate the same model as your pretrained models?

**Comment by nreimers** [comment]
Code:
https://www.sbert.net/examples/training/multilingual/README.html

Datasets:
https://huggingface.co/datasets/sentence-transformers/parallel-sentences

**Comment by jzju** [comment]
Thanks but I would hope to get the exact replica. Like what learning rate, iterations and batch size they used.
